interface/src/app/gene/network/network.js

```diff
@@ -134,22 +136,46 @@ angular.module('adage.gene.network', [
       };
 
       self.slider = {  // range slider configuration
-        min: 0,                    // initial position of slider on the left
-        max: maxCorrelation,       // initial position of slider on the right
+        min: 0,                  // initial position of slider on the left
+        max: maxCorrelation,     // initial position of slider on the right
         options: {
-          floor: 0,                // minimum of the slider bar
-          ceil: maxCorrelation,    // maximum of the slider bar
+          floor: 0,              // minimum of the slider bar
+          ceil: maxCorrelation,  // maximum of the slider bar
           step: 0.01,
           precision: 2,
-          showTicks: 0.5,
-          showTicksValues: true,
           noSwitching: true,
           onEnd: function(id, low, high) {
             self.renderNetwork();
           }
         }
       };
 
+      // After machine learning model in the URL has been validated,
+      // reconfigure the slider and force render it. See the discussion at:
+      // https://github.com/angular-slider/angularjs-slider/issues/79#issuecomment-219213647
+      // (Also tried $scope.$$postDigest, not work.)
+      var refreshSlider = function() {
+        $timeout(function() {
+          $scope.$broadcast('rzSliderForceRender');
+        }, 250);
+        // Note by dhu: 250ms is an arbitrary timeout value. The minimum timeout
+        // value that worked on my desktop is 150ms.
+      };
```

Comment:
Actually the most relevant ref URL is this one:
https://github.com/angular-slider/angularjs-slider/issues/79#issuecomment-225438841